src/lib/parse/compoundValue.cpp

```diff
@@ -162,6 +157,8 @@ void compoundValueEnd(ConnectionInfo* ciP, ParseData* parseDataP)
   // latest contextAttribute, i.e. the attribute whose 'contextValue' is the
   // owner of this compound value tree.
   //
+  // FIXME P4 #1862: obsolete reference to XML in the above comment?
```

Comment:
@kzangeli I'd need the new version of the above comment (or maybe it can be removed)?